libbeat/docs/loggingconfig.asciidoc

```diff
@@ -107,37 +107,39 @@ metrics and for this reason they are also not documented.
 
 
 [float]
-==== `metrics.period`
+==== `logging.metrics.period`
 
 The period after which to log the internal metrics. The default is 30s.
 
 [float]
-==== `files.path`
+==== `logging.files.path`
 
 The directory that log files are written to. The default is the logs path. See the
 <<directory-layout>> section for details.
 
 [float]
-==== `files.name`
+==== `logging.files.name`
 
 The name of the file that logs are written to. By default, the name of the Beat
 is used.
 
 [float]
-==== `files.rotateeverybytes`
+==== `logging.files.rotateeverybytes`
 
 The maximum size of a log file. If the limit is reached, a new log file is generated.
 The default size limit is 10485760 (10 MB).
 
 [float]
-==== `files.keepfiles`
+==== `logging.files.keepfiles`
 
 The number of most recent rotated log files to keep on disk. Older files are
 deleted during log rotation. The default value is 7. The `keepfiles` options has to be
 in the range of 2 to 1024 files.
 
 [float]
-==== `files.permissions`
+==== `logging.files.permissions`
+
+//REVIEWERS: Noticed that his is not in the beat.reference.yml file. Should it be? Or is the option old?
```

Comment:
Yeah, it should be in there too. Can you please add it to [config.reference.yml](https://github.com/elastic/beats/blob/e1060ff722e8f68691ed646599dda575945e5efa/libbeat/_meta/config.reference.yml#L736-L749) and run `make update` from the top-level?

Reply:
@andrewkroh Added. Can you review the changes? Thanks!